src/includes/class_reservationPermissions.php

```diff
@@ -413,10 +467,16 @@ public function insertCSVFile() {
     }
   }
 
+  // multiDelete
+  // ==================================================
+  // Name : Tracy McCormick
+  //
+  // Description:
+  // Function performs a delete on all records passed from the form
   public function multiDelete($items = null){
     try{
-  		foreach ($items as $reservationID){
-  			self::deleteRecord($reservationID);
+  		foreach ($items as $id){
+  			$this->deleteRecord($id);
```

Comment:
Yes be more descriptive, however, slitting hairs here.  This won't hold back the pull request.